packages/react/src/primitives/Pagination/PaginationItem.tsx

```diff
@@ -60,7 +60,8 @@ export const PaginationItem: React.FC<PaginationItemProps> = ({
         <View as="li">
           {page === currentPage ? (
             <Flex
-              as="span"
+              aria-current="true"
```

Comment:
@hbuchel Should this be set to `aria-current="page"`? 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#values

Reply:
Oops sorry, yeah that is better, thanks! @joebuono there is a comment we can remove just below it too that references the menu aria pattern that isn't relevant anymore.